### DISCUSSION TITLE
Starten des Cruiser Follow on Modes durch langes Drücken des Wonderwheels schon verfügbar?

### DISCUSSION THREAD

**Discussion by RolfGS** [devemux86/cruiser#633] [Q&A] [answered]
Ist in der aktuellen Cruiser Version das Starten des Follow on Modes durch langes Drücken des Wonderwheels schon verfügbar?

**Comment by devemux86** [comment]
@RolfGS 

Which device do you mean? Do you have a link?
I added it for WunderLINQ because they publish documentation so it was easy to implement.

Welches Gerät meinen Sie? Haben Sie einen Link?
Ich habe es für WunderLINQ hinzugefügt, weil sie Dokumentation veröffentlichen, so dass es einfach zu implementieren war.

- https://github.com/devemux86/cruiser/discussions/632

**Reply by RolfGS** [reply]
Hallo,
ich dachte dass ich diese Funktion bekomme wenn ich die neueste Cruiser Version herunterladen. Ist das nicht so ?
Muss ich auch WunderLINQ Version updaten und auch die Cruiser Version ?
Sorry für die naiven Fragen.
Gruss,
Rolf

**Reply by devemux86** [reply, marked as answer]
@RolfGS 

You need to use Cruiser 5 (Beta) from here:
Or can wait its public release the next days.

Sie müssen Cruiser 5 (Beta) von hier verwenden:
Oder man kann seine Veröffentlichung in den nächsten Tagen abwarten.

- https://github.com/devemux86/cruiser/discussions/610

**Reply by RolfGS** [reply]
Oh, danke, also "nur" Cruiser Beta laden.
Wird Cruiser Beta 5.0.0 oder Cruiser 5.0.0 in den nächsten Tagen veröffentlicht ?

...ich nutze Cruiser zwar nicht so viel wie einige User hier, bin aber in 2024 zum Cruiser Fan geworden--planen in Kurviger am PC--> Google Drive--> Cruiser ist für mich die beste Tourenplanung die ich je hatte ! Danke !

**Reply by devemux86** [reply]
@RolfGS 

Thanks for the kind words!
Cruiser 5 is currently in final testing, it will be released soon.

Danke für die freundlichen Worte!
Cruiser 5 ist derzeit in den letzten Tests, es wird bald veröffentlicht werden.

**Reply by RolfGS** [reply]
Das Kompliment ist ernst gemeint!
Ich habe momentan Cruiser 4.0.2 auf dem Smartphone und in Playstore wird mir keine aktueller Version angeboten (klar die 5.0.0 ist noch nicht da) . Wie kann ich eine aktuellere freigegebene Version von Cruiser (also zwischen 4.0.2 und 5.0.0) laden ?

**Reply by devemux86** [reply]
@RolfGS 

Cruiser 5 on Google Play will be released in the near future.
Now there is Cruiser 5 (beta), you can install the apk manually after you backup your favorites, routes, etc.
If you are not comfortable with beta testing, better test it on another phone or wait for the release on Google Play.

Cruiser 5 auf Google Play wird in naher Zukunft veröffentlicht werden.
Jetzt gibt es Cruiser 5 (Beta), können Sie die apk manuell installieren, nachdem Sie Ihre Favoriten, Routen, etc. gesichert haben.
Wenn Sie sich mit Beta-Tests nicht wohlfühlen, testen Sie es besser auf einem anderen Telefon oder warten Sie auf die Veröffentlichung auf Google Play.

- https://github.com/devemux86/cruiser/discussions/610

**Reply by RolfGS** [reply]
Hallo, danke, ich warte auf Cruiser 5 auf Google Play weil ich mom wenig Anwendung habe. Aber ich wechsele gerade von Samsung S10e auf S24+ , da werde ich sicher erst Cruiser 5 verwenden. Wie lange wird das noch ca dauern? eher 3 Monate oder ???...nur ungefähr...und unverbindlich...

**Reply by devemux86** [reply]
@RolfGS 

It will be very soon. :slightly_smiling_face: 

Es wird sehr bald sein. :slightly_smiling_face:

**Reply by RolfGS** [reply]
ok, thanks, will stay paitient...
